docs/book/user-guide/component-guide/data-validators/deepchecks.md

```diff
@@ -126,85 +126,77 @@ To instantiate a data integrity step that will run all available Deepchecks data
-
-    def entrypoint(  # type: ignore[override]
-            self,
-            dataset: pd.DataFrame,
-            params: DeepchecksDataIntegrityCheckStepParameters,
-    ) -> SuiteResult:
-        ...
+@step
+def deepchecks_data_integrity_check_step(
+    dataset: pd.DataFrame,
+    check_list: Optional[Sequence[DeepchecksDataIntegrityCheck]] = None,
+    dataset_kwargs: Optional[Dict[str, Any]] = None,
+    check_kwargs: Optional[Dict[str, Any]] = None,
+    run_kwargs: Optional[Dict[str, Any]] = None,
+) -> SuiteResult:
+    ...
 ```
 
-If needed, you can specify a custom list of data integrity Deepchecks tests to be executed by supplying a `check_list` argument to the step configuration:
+If needed, you can specify a custom list of data integrity Deepchecks tests to be executed by supplying a `check_list` argument:
 
 ```python
 from zenml.integrations.deepchecks.validation_checks import DeepchecksDataIntegrityCheck
-from zenml.integrations.deepchecks.steps import (
-    DeepchecksDataIntegrityCheckStepParameters,
-    deepchecks_data_integrity_check_step,
-)
+from zenml.integrations.deepchecks.steps import deepchecks_data_integrity_check_step
+
 
-data_validator = deepchecks_data_integrity_check_step(
-    step_name="data_validator",
-    params=DeepchecksDataIntegrityCheckStepParameters(
+@pipeline
+def validation_pipeline():
+    deepchecks_data_integrity_check_step(
         check_list=[
             DeepchecksDataIntegrityCheck.TABULAR_MIXED_DATA_TYPES,
             DeepchecksDataIntegrityCheck.TABULAR_DATA_DUPLICATES,
             DeepchecksDataIntegrityCheck.TABULAR_CONFLICTING_LABELS,
         ],
-    ),
-)
+        dataset=...
+    )
 ```
 
 You should consult [the official Deepchecks documentation](https://docs.deepchecks.com/en/stable/checks\_gallery/tabular.html) for more information on what each test is useful for.
 
-For more customization, the `DeepchecksDataIntegrityCheckStepParams` step configuration also allows for additional keyword arguments to be supplied to be passed transparently to the Deepchecks library:
+For more customization, the data integrity step step also allows for additional keyword arguments to be supplied to be passed transparently to the Deepchecks library:
 
 *   `dataset_kwargs`: Additional keyword arguments to be passed to the Deepchecks `tabular.Dataset` or `vision.VisionData` constructor. This is used to pass additional information about how the data is structured, e.g.:
 
     ```python
-    params=DeepchecksDataIntegrityCheckStepParameters(
-        dataset_kwargs=dict(label='class', cat_features=['country', 'state']),
-    ),
+    deepchecks_data_integrity_check_step(
+        dataset_kwargs=...,
+        ...
+    )
     ```
 *   `check_kwargs`: Additional keyword arguments to be passed to the Deepchecks check object constructors. Arguments are grouped for each check and indexed using the full check class name or check enum value as dictionary keys, e.g.:
```

Comment:
This is all red in GitHub, feels like the syntax is broken somewhere in this file?